src/plugins/inputs/cycle.js

```diff
@@ -0,0 +1,31 @@
+"use strict";
```

Comment:
I understand where you're coming from, and I don't disagree with the rationale in terms of usage which is why I'm happy with a `/cycle` alias, but when it comes to the filenames and such, this is The Lounge codebase territory.
We don't have other command-related filenames with `de`/`re` suffixes yet, but we do have commands that have their counterparts prefixed with `de` (`voice` vs. `devoice`, etc.). This file should then be `rejoin.js`, with support for `cycle` command.